.chloggen/update-default-traces-exporter-to-otlphttp.yaml

```diff
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: operator
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: update default traces exporter to otlphttp
+# One or more tracking issues related to the change
+issues: [1518]
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Comment:
We should add some breaking change migration notes here or in the [UPGRADING.md](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/UPGRADING.md) file. I might suggest something like this.

> If you use the sapm exporter with custom settings, you have two options:
> - Migrate your sapm settings to the new otlphttp exporter.
> - Retain SAPM settings by moving them to your agent.config or gateway.config overrides to ensure they remain effective.